src/WebJobs.Extensions.DurableTask.Analyzers/Analyzers/Orchestrator/MethodAttributeAnalyzer.cs

```diff
@@ -53,7 +53,7 @@ private void AnalyzeMethod(SyntaxNodeAnalysisContext context)
 
             var declaration = syntaxReference.GetSyntax(context.CancellationToken);
 
-            if (SyntaxNodeUtils.IsMarkedDeterministic(declaration))
+            if (SyntaxNodeUtils.IsMarkedDeterministic(declaration) || SyntaxNodeUtils.IsInsideOrchestrator(declaration))
```

Comment:
Just to clarify, IsInsideOrchestrator() will return `true` for a local function that is nested inside of an orchestrator function, right?

Reply:
Correct. In the case that this is true, it will not create the diagnostic. So when there is a local function inside an orchestrator, it will already be analyzed and there is no need for the Deterministic attribute. This is a temporary fix to allow this case to work smoothly until I refactor this analyzer to not look for Deterministic attributes at all.